WordPress/gutenberg
# [RNMobile] Add embed webview for Android

## What?
Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/5509

Adds a full screen embed preview for Android.

Depends on:
- https://github.com/wordpress-mobile/WordPress-Android/pull/18383
- https://github.com/wordpress-mobile/gutenberg-mobile/pull/5743

Related:
- https://github.com/Automattic/jetpack/pull/30521




## Why?

The `react-native-webview`  on Android has event propagation issues when nested inside a `Pressable` component. As a work around we need an Android WebView to render interactive embed content.

## How?

Adds a new Android Activity to render embed content in a native WebView.

## Testing Instructions




- Checkout the related Jetpack PR
- Open a post on the Android Jetpack app 
- Add or view a VideoPress block in the editor
- Select the block and press the video.
- A native WebView should open.
- Verify that the embed content responds to touch events as expected.
- Close the WebView by swiping or pressing the back arrow
- The app should return to the editor.


### Testing Instructions for Keyboard


## Screenshots or screencast
